### DISCUSSION TITLE
Why GitHub is releasing so many achievements?

### DISCUSSION THREAD

**Discussion by Kai-Wang0** [community/community#20579] [Other Feature Feedback, Questions, & Ideas] [answered]
I can see a lot of new achievements on many of my friend's profiles. Why is GitHub launching them?

**Comment by Mehul2203** [comment, marked as answer]
I think GitHub created this achievement just to show how quickly we respond to any PR. 🚀
They are trying to get all of the GitHub users to get more active and to keep us motivated to work with team dynamics in an effective manner.

**Comment by ghost** [comment]
Besides the benefit of its existence as mentioned @Mehul2203 
  But it made people do nothing (like faked Q/A) for the sake of achievement

**Reply by Mehul2203** [reply]
Yes, I agree.

**Comment by topstarwebking** [comment]
For example, if developer have more achievements than other developers, it's high to get hired from clients.
